cmd/export_contract_events_test.go

```diff
@@ -0,0 +1,20 @@
+package cmd
+
+import (
+	"testing"
+)
+
+func TestExportContractEvents(t *testing.T) {
+	tests := []cliTest{
+		{
+			name:    "diagnostic events from one ledger",
+			args:    []string{"export_contract_events", "-s", "52271338", "-e", "52271350", "-o", gotTestDir(t, "one_ledger_txs.txt")},
```

Comment:
> is there only one ledger written to the file? The export range is larger than 1

Is this because data is coming from test hubble?

> Also file name is a bit confusing if the output data is related to contract_events

Agree. Will rename it.